easybuild/easyconfigs/s/structure/structure-2.3.4-GCC-6.3.0.eb

```diff
@@ -0,0 +1,34 @@
+#
+# This file is an EasyBuild reciPy by Christian Meesters (2017)
+# HPC-group University of Mainz (https://hpc.uni-mainz.de)
+
+easyblock = 'MakeCp'
+
+name = 'structure'
+version = '2.3.4'
+
+homepage = 'https://web.stanford.edu/group/pritchardlab/structure.html'
+description = """The program structure is a free software package for using multi-locus
+ genotype data to investigate population structure. Its uses include infering 
+ the presence of distinct populations, assigning individuals to populations, 
+ studying hybrid zones, identifying migrants and admixed individuals, and estimating
+ population allele frequencies in situations where many individuals are migrants or admixed. 
+ It can be applied to most of the commonly-used genetic markers, including SNPS, 
+ microsatellites, RFLPs and AFLPs."""
+
+toolchain = {'name': 'GCC', 'version': '6.3.0'}
```

Comment:
This toolchain doesn't exist, you should either use `GCCcore/6.3.0` or `GCC/6.3.0-2.28` (I strongly recommend the latter, since that toolchain already includes `binutils`, otherwise you need to list `binutils` yourself as a build dependency)